CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+* Tweak `BakeExample` to not touch unbaked titles in captions (major)
```

Comment:
this should be minor (or patch?) since, although it is changing the implementation of `BakeExample`, it's just a bug fix & not something that will like. impact or break other books